zmq-sys/build.rs

```diff
@@ -3,6 +3,27 @@ extern crate metadeps;
 use std::env;
 use std::path::Path;
 
+fn verify_windows_build_environment() {
+    let target = env::var("TARGET").ok().unwrap();
+    println!("cargo:target={:?}", target);
+    if target.contains("x86_64-pc-windows") {
+        let dll_path = prefix_dir("LIBZMQ_BIN_DIR", "bin");
+        match &dll_path {
+            Some(dll_path) => {
+                println!("cargo:dll_path={}", &dll_path);
```

Comment:
Likewise, I couldn't dig up any documentation about `cargo:dll_path`.

Reply:
I just used the reference to `cargo:dll_path` to print some meaningful information for the user in the event that the code panics.
Will remove `fn verify_windows_build_environment()` altogether as suggested, so this will be resolved.